tests/test_lib_events.py

```diff
@@ -1467,6 +1467,13 @@ def test_05_check_conditions(self):
              "request": req})
         self.assertEqual(r, False)
 
+        r = uhandler.check_condition(
+            {"g": {},
+             "handler_def": {"conditions": {"result_status": True}},
```

Comment:
Shouldn't this be ``{"result_status": "True"}``? i.e. the event condition should contain strings, not booleans (same with the ``result_value`` test above).

Reply:
The code will receive a boolean ``True`` since this is the data privacyIDEA generates and not that the API returns.
See line https://github.com/privacyidea/privacyidea/pull/972/files#diff-bf9bddc6356fca28b236be606b262187L1465

Reply:
Hmm, but if I read the code correctly, privacyIDEA does not actually generate a boolean ``True``, but a string ``"True"``, because the ``CONDITION.RESULT_STATUS`` and ``CONDITION.RESULT_VALUE`` conditions are defined as a choice between ``"True"`` and ``"False``", of type ``str``?
```python
            CONDITION.RESULT_STATUS: {
                "type": "str",
                "desc": _("The result.status within the response is "
                          "True or False."),
                "value": ("True", "False")
            },
```

Reply:
You are right. The "conditions-value" is read from the database table ``eventhandlercondition`` and is always a string representation.